addons/web_editor/static/lib/odoo-editor/src/commands/deleteBackward.js

```diff
@@ -74,6 +74,10 @@ HTMLElement.prototype.oDeleteBackward = function (offset, alreadyMoved = false)
             leftNode.remove();
             return;
         }
+        if (leftNode.getAttribute && leftNode.getAttribute('contenteditable') === 'false') {
```

Comment:
I would just put that in a `||` of the previous check since it does the exact same thing.
Also, wouldn't this fail for `<my-node contenteditable="0">` and `<my-node contenteditable="False">` for instance? Aren't those valid forms?